### PR TITLE
minor edits to doco for plugins

### PR DESCRIPTION
**Pull request by pacharanero**
I was passing anyway while looking something up....

* edits to code blocks using newer markdown and python specific syntax
* indentation fixes in code blocks
* more detail on Installing plugins

### CODE REVIEW

File: doc/docs/guides/plugins.md [hunk 1]
```diff
@@ -7,8 +7,9 @@ models for the client.
 
 The Opal commandline tool will bootstrap your plugin for you - just run:
 
-    $ opal startplugin yourcoolplugin
-
+```python
+$ opal startplugin yourcoolplugin
```
Comment (fredkingham) [inline doc/docs/guides/plugins.md]: this is shell script

File: doc/docs/guides/plugins.md [hunk 2]
```diff
@@ -142,8 +148,9 @@ class YourPlugin(OpalPlugin):
 
 ### Installing plugins
 
-Add to installed apps
-Add to requirements if appropriate
+* Add the plugin'a name to the `INSTALLED_APPS` list in your `settings.py`
```
Comment (fredkingham) [inline doc/docs/guides/plugins.md]: plugin's

File: doc/docs/guides/plugins.md (hunk 1 above)
Comment (pacharanero) [reply doc/docs/guides/plugins.md]: Doh! 

fixed

File: doc/docs/guides/plugins.md (hunk 2 above)
Comment (pacharanero) [reply doc/docs/guides/plugins.md]: thanks

fixed

**Comment by pacharanero** [comment]
Those errors now fixed

**Pull request merged**